SPECS/kernel/config

```diff
@@ -7374,7 +7375,6 @@ CONFIG_IPE_PROP_DM_VERITY=y
 CONFIG_IPE_PROP_DM_VERITY_SIGNATURE=y
 CONFIG_IPE_PROP_FS_VERITY=y
 CONFIG_IPE_PROP_FS_VERITY_BUILTIN_SIG=y
-CONFIG_IPE_PROP_INTENDED_PATHNAME=y
```

Comment:
"IPE_PROP_INTENDED_PATHNAME" is no longer included in the AzL kernel ipe source to better align with upstream offerings. Commit was removed in feature branch: microsoft/CBL-Mariner-Linux-Kernel@f87bb12

for reference, commit which contained this feature was [d356a4ff68] ipe: add intended_pathname property